docs/contributing.md

```diff
@@ -6,24 +6,56 @@ The following are guidelines for contributing to this project.
+This repository is governed by Mozilla's code of conduct and etiquette
+guidelines. For more details please see the [Mozilla Community Participation
+Guidelines][community-guidelines] and [Developer Etiquette
+Guidelines][etiquette-guidelines].
 
 ## How to Get Started
 
-Please refer to installation and build instructions in the [install documentation](install.md).
+Please refer to installation and build instructions in the [install
+documentation](install.md).
 
 ## How to Report Bugs
 
-Please open [a new issue in the appropriate GitHub repository](https://github.com/mozilla-lockbox/lockbox-extension/issues/new) with steps to reproduce the problem you're experiencing.
+Please open [a new issue in the appropriate GitHub repository][new-issue] with
+steps to reproduce the problem you're experiencing.
 
-Be sure to include as much information including screenshots, text output, and both your expected and actual results.
+Be sure to include as much information including screenshots, text output, and
+both your expected and actual results.
 
 ## How to Request Enhancements
 
-First, please refer to the applicable [GitHub repository](https://github.com/orgs/mozilla-lockbox/) and search [the repository's GitHub issues](https://github.com/mozilla-lockbox/lockbox-extension/issues) to make sure your idea has not been (or is not still) considered. It may also be easier to see and search across all projects combined on our [Waffle.io planning board](https://waffle.io/mozilla-lockbox/lockbox-extension).
+First, please refer to the applicable [GitHub repository][github-repos] and
+search [the repository's GitHub issues][issues-list] to make sure your idea has
+not been (or is not still) considered. It may also be easier to see and search
+across all projects combined on our [Waffle.io planning board][waffle].
 
-Then, please [create a new issue in the GitHub repository](https://github.com/mozilla-lockbox/lockbox-extension/issues/new) describing your enhancement.
+Then, please [create a new issue in the GitHub repository][new-issue] describing
+your enhancement.
 
-Be sure to include as much detail as possible including step-by-step descriptions, specific examples, screenshots or mockups, and reasoning for why the enhancement might be worthwhile.
+Be sure to include as much detail as possible including step-by-step
+descriptions, specific examples, screenshots or mockups, and reasoning for why
+the enhancement might be worthwhile.
 
-Please keep in mind, by opening an issue we provide no guarantee the enhancement will be implemented.
+Please keep in mind, by opening an issue we provide no guarantee the enhancement
+will be implemented.
+
+## How to Contribute Code
+
+Before you get started writing code, be sure what you plan to work on is
+something we'll be able to accept. The easiest way is to look through out list
+of [good first issues][good-first-issues] and find something that sounds
```

Comment:
Is this “first issues” link defined? I don’t see it anywhere in this doc...

Reply:
I think I forgot that. I'll add it in.

Reply:
Ref: https://github.com/mozilla-lockbox/lockbox-extension/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22